app/views/content/life-as-a-teacher/chemistry/_article.html.erb

```diff
@@ -0,0 +1,127 @@
+    <p>There are lots of reasons why someone might need to complete an SKE course. For example, if you finished your degree more than 5 years ago or if you have an A level but not a degree in the subject you want to teach.</p>
+    <p>Find out more about <a href="/how-to-apply-for-teacher-training/subject-knowledge-enhancement">subject knowledge enhancement courses</a>.</p>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h3>Fund your teacher training</h3>
+    <p><a href="/funding-and-support/scholarships-and-bursaries"> Tax-free bursaries of <%= v :bursaries_postgraduate_chemistry %> or scholarships of <%= v :scholarships_chemistry %></a> are available for eligible trainee chemistry teachers.</p>
+    <p>You can get a bursary or scholarship alongside a tuition fee and maintenance loan. Even if you've had financial support for a previous course, you could still be eligible for funding to support your teacher training.</p>
+    <p>You may also be able to get <a href="/funding-and-support">extra funding and support</a> if you're:</p>
+    <ul>
+      <li>a parent or carer</li>
+      <li>disabled</li>
+      <li>a veteran</li>
+    </ul>
+    <%= render Content::ExpanderComponent.new(
+    title: "funding your training",
+    text: "If you do not have <a href=\"https://www.gov.uk/guidance/indefinite-leave-to-remain-in-the-uk\">indefinite leave to remain in the UK</a>, you're unlikely to be eligible for funding.",
+    link_title: "Find out more about fees and financial support for non-UK trainee teachers.",
+    link_url: "/non-uk-teachers/fees-and-funding-for-non-uk-trainees",
+  ) %>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h3>Apply for a paid internship</h3>
+    <p>If you're currently doing an undergraduate or master's degree and are interested in teaching chemistry, you could apply for a <a href="/train-to-be-a-teacher/teaching-internships">paid teaching internship</a>.</p>
+    <p>The 3-week programme could help you to understand what it's really like in the classroom and get a feel for school life.</p>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Support with teacher training</h2>
+    <p>Throughout your initial teacher training in chemistry, you'll have the support of a mentor.</p>
+    <p>Your mentor will:</p>
+    <ul>
+      <li>guide you through your training, sharing learning and career development opportunities with you</li>
+      <li>meet with you regularly to make sure your training is a positive experience</li>
+    </ul>
+    <p>You'll also be part of a welcoming school community where you'll learn from fellow teachers through taught sessions, seminars and workshops.</p>
+    <p>Find out more about what support you'll get as <a href="/support-for-early-career-teachers">an early career teacher</a>.</p>
```

Comment:
I think this is a bit confusing - ECT comes **after** ITT

You could add a sentence along the lines of - "And the support continues during your first 2 years of teaching. Find out about the support you'll get as an early career teacher'